### DISCUSSION TITLE
How to properly expose ports on milvus running with k3s (with helm)?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#22889] [Q&A and General discussion]
How to properly expose ports on milvus running with k3s (with helm)?
Does the load automatically will be distributed across all nodes?

**Comment by yhmo** [comment]
Use `kubectl port-forward `to expose ports
https://milvus.io/docs/install_cluster-helm.md#Connect-to-Milvus
